.evergreen/scripts/compile-unix.sh

```diff
@@ -208,12 +210,12 @@ fi
 . "${script_dir:?}/find-ccache.sh"
 find_ccache_and_export_vars "$(pwd)" || true
 
-"${cmake_binary}" "${configure_flags[@]}" ${extra_configure_flags[@]+"${extra_configure_flags[@]}"} .
-"${cmake_binary}" --build .
-"${cmake_binary}" --install .
+"${cmake_binary}" -S . -B build "${configure_flags[@]}" ${extra_configure_flags[@]+"${extra_configure_flags[@]}"} .
```

Comment:
```suggestion
"${cmake_binary}" -S . -B cmake-build "${configure_flags[@]}" ${extra_configure_flags[@]+"${extra_configure_flags[@]}"} .
```

Consider using `cmake-build` to avoid possibly interfering with existing files in the `build` directory.